dashboard/app/views/levels/_unplug.html.haml

```diff
@@ -1,5 +1,5 @@
 - unplugged = @level.type == 'Unplugged' ? @level.name : @game.name
-- video_key = try_t("data.unplugged.#{unplugged}.video")
+- video_key = @level.video_key || try_t("data.unplugged.#{unplugged}.video")
```

Comment:
Oh, wow, I didn't know we did this. I would consider this current behavior broken and we should fix it so that in non-EN we still show the English video (with translated captions). I'm in favor of moving unplugged levels to .level files and associating the video from there.

We only have a few localized videos so it's not worth it right now, but in the future we'd like to build something where we could substitute the EN video with a localized video. In which case I would agree with Josh that it should depend on video_key not a translated video name.